Jenkinsfile

```diff
@@ -114,6 +112,34 @@ pipeline {
           sh 'rm -rf examples/nlp/lightning_logs'
         }
     }
+    stage('L3: NER') {
+      when {
+        anyOf{
+          branch 'candidate'
+          changeRequest()
+        }
+      }
+      failFast true
+        steps {
+          sh 'cd examples/nlp/token_classification && python ner.py \
```

Comment:
Shouldn't we limit the GPU by setting CUDA_VISIBLE_DEVICES here?

Reply:
I thought CUDA_VISIBLE_DEVICES doesn't work with PL and we can explicitly specify gpu in trainer. Let's leave it as is for now, we. would need to refactor tests anyway

Reply:
I think Oleksii mentioned in another PR and we should have it when we have parallel tests to isolate them.

Reply:
What happens when two test are run on single gpu? Can pt handle such cases? We can now leave it like this for now and check again later.